src/js/protocols/stm32.js

```diff
@@ -422,31 +423,31 @@ STM32_protocol.prototype.verify_chip_signature = function (signature) {
 // second_array = usually verify_hex array
 // result = true/false
 STM32_protocol.prototype.verify_flash = function (first_array, second_array) {
-    for (var i = 0; i < first_array.length; i++) {
+    for (let i = 0; i < first_array.length; i++) {
```

Comment:
Did you want to fix `first_array` as well?

Reply:
Renamed